recipes/py2exe/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "py2exe" %}
+{% set version = "0.11.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/py2exe/py2exe/archive/v{{ version }}.tar.gz
+  sha256: 6bb7ff4834ef2096823710c67edf49c971422775fa41c9c7718c19f0437a2d3d
+
+build:
+  number: 0
+  entry_points:
+    - build_exe = py2exe.build_exe:main
+  skip: True  # [not win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
```

Also we need this to setup the C compiler

```suggestion
requirements:
  build:
    - {{ compiler('c') }}
```